requirements.txt

```diff
@@ -2,10 +2,10 @@ nltk
 pandas
 PyYAML
 scikit-learn
-torch==1.8.0
-torchmetrics==0.6.0
-torchtext==0.9.0
-pytorch-lightning==1.3.5
+torch
```

Comment:
Should we leave the version numbers here? Sometimes the installation failed due to torchtext update, it changes a lot between versions (Ex. from 0.8.0 to 0.9.0).

Reply:
Sure, thanks for pointing out. Users can also know the actual version here.